_docs/5_SubspaceNetwork/README.md

```diff
@@ -263,3 +263,21 @@ docker-compose up -d
 
 Typical commands like `docker-compose restart`, `docker-compose logs --tail=100 -f` can be used to manage this setup,
 see [Docker Compose docs](https://docs.docker.com/compose/reference/) for details.
+
+### Datadog Integration
```

Comment:
It is already described in `_docs/4_Docker/README.md` file why do we need it here too?